synapse/tests/test_lib_storm.py

```diff
@@ -4816,3 +4512,34 @@ async def test_lib_dmon_embed(self):
             ''')
 
             self.none(await core.callStorm('return($lib.queue.gen(haha).get().1)'))
+
+    # async def test_storm_onload_persistence(self):
```

Comment:
Is this commented on purpose or an artifact that you forgot to uncomment/delete?